kernal/cbm/editor.s

```diff
@@ -261,13 +265,33 @@ loop3
 	and #(255-MODIFIER_4080)
 	sta shflag
 	
-	sec             ;Get current screen mode
-	jsr screen_mode
-
-	eor #SCREEN_MODE_4030
-	clc
-:	jsr screen_mode ;Set new screen mode, will clear screen
-
+;	sec             ;Get current screen mode
```

Comment:
Consider removing the code that's commented out

Reply:
Good point, I overlooked this before creating the PR.